### ISSUE TITLE
Handling hunter package manager

### ISSUE THREAD

**Issue by rnhmjoj** [NixOS/nixpkgs#27431]
## Issue description
I have come across a [program](https://github.com/Genoil/cpp-ethereum) using the [hunter](https://docs.hunter.sh/en/latest/) package manager and I have no idea how to package it. Hunter is designed to [ignore](https://docs.hunter.sh/en/latest/faq/interaction-with-other-package-managers.html) any system-installed package and instead fetch them on its own.

it seem no hunter program has been package before in nixpkgs.

**Comment by emmanuelrosa** [comment]
Correct, as far as I know there's nothing in Nixpkgs to handle hunter-based builds. But there's nothing stopping you from creating one :)

Take a look at how other build tools and package managers are handled here: https://nixos.org/nixpkgs/manual/#chap-language-support

**Comment by ob7** [comment]
did you ever manage to use hunter package manager in nixos?  I'm trying to package https://github.com/ethereum-mining/ethminer and it requires hunter.

**Comment by rnhmjoj** [comment]
No, I didn't really try but I think it's beyond my skills.

**Comment by siraben** [comment]
still important to me

**Comment by siraben** [comment]
Related: #83554

**Comment by hacker1024** [comment]
still important to me

**Comment by Artturin** [comment]
At some point ethminer added a flag to disable hunter https://github.com/ethereum-mining/ethminer/blob/master/docs/BUILD.md#disable-hunter
all projects which use hunter should probably add that, unless hunter has added a flag or something to use system dependencies, otherwise projects using hunter can't be packaged without a FOD.

**Comment by samueldr** [comment]
While triaging, I'm noting here that the domain name lapsed.

Some notes here:

 - https://www.fltk.org/str.php?L3505 (comment `#`7)

The new(?) project is reportedly maintained(?) here:

 - https://github.com/cpp-pm/hunter
